openc3/lib/openc3/script/limits.rb

```diff
@@ -23,17 +23,18 @@ module Script
 
     # Define all the modification methods such that we can disconnect them
     %i(enable_limits disable_limits set_limits enable_limits_group disable_limits_group set_limits_set).each do |method_name|
-      define_method(method_name) do |*args|
+      define_method(method_name) do |*args, **kw_args, &block|
```

Comment:
Do any of these actually take a block?

Reply:
Don't know.  But if they do it will work now.